examples/drawings.js

```diff
@@ -32,6 +32,24 @@ const regionToColor = region => {
   }
 };
 
+const selectedCategory = 'teapot';
+const categories = [
+  'flower',
+  'piano',
+  'power outlet',
+  'teapot',
+  'The Mona Lisa',
+  'The Great Wall of China',
```

Comment:
Let's not include those as there wasn't much to see. If you want to keep the great wall in for sentimental reasons please do so :)